### PR TITLE
Interval search

### PR DESCRIPTION
**Pull request by stefanks**

### CODE REVIEW

File: Morpheus/Morpheus/command line/Program.cs [hunk 1]
```diff
@@ -198,6 +183,9 @@ static void Main(string[] args)
                     output_folder = arguments["o"];
                 }
 
+                List<double> massErrors = new List<double>();
+                massErrors.Add(0);
```
Comment (cwenger) [inline Morpheus/Morpheus/command line/Program.cs]: Can you add a command-line option for mass errors?

File: Morpheus/Morpheus/frmMain.cs [hunk 2]
```diff
@@ -171,18 +171,6 @@ private void frmMain_Load(object sender, EventArgs e)
...
-                                    {
-                                        chkPrecursorMonoisotopicPeakCorrection.Checked = bool.Parse(value);
-                                    }
-                                    break;
-                                case "Minimum Precursor Offset":
-                                    numMinPrecursorMonoPeakOffset.Value = int.Parse(value);
-                                    break;
-                                case "Maximum Precursor Offset":
-                                    numMaxPrecursorMonoPeakOffset.Value = int.Parse(value);
-                                    break;
```
Comment (cwenger) [inline Morpheus/Morpheus/frmMain.cs:185]: Did you choose to leave out restoring of the mass errors setting intentionally?

File: Morpheus/Morpheus/command line/Program.cs (hunk 1 above)
Comment (stefanks) [inline Morpheus/Morpheus/command line/Program.cs]: Done

File: Morpheus/Morpheus/frmMain.cs (hunk 2 above)
Comment (stefanks) [inline Morpheus/Morpheus/frmMain.cs:185]: No, I'll look into it right now

Comment (stefanks) [inline Morpheus/Morpheus/frmMain.cs:185]: Fixed

**Pull request merged**